docs/tactics.md

```diff
@@ -1186,7 +1186,7 @@ begin
 end
 ```
 
-Although `reassoc_of` is not a tactic or a meta program, its type is generated 
+Although `reassoc_of` is not a tactic or a meta program, its type is generated
```

Comment:
@cipher1024, I think you're often committing changes with trailing whitespaces, that then result in spurious diffs as other people edit these files. Would you mind enabling `"files.trimTrailingWhitespace": true` if you use VS Code?

Reply:
Unfortunately, @cipher1024 is a hardcore emacs user :smiley_cat: But I'm sure there is an emacs equivalent...

Reply:
yes: `(add-hook 'before-save-hook 'delete-trailing-whitespace)`

Reply:
I have to apologize. I have that setting but often, I use the github interface to write missing documentation. Is there a setting for github to trim the spaces?

Reply:
https://stackoverflow.com/questions/591923/make-git-automatically-remove-trailing-whitespace-before-committing